### PR TITLE
bpo-42392: Remove deprecated loop parameter from docs

### PR DESCRIPTION
**Pull request by uriyyo**
Remove deprecated *loop* parameter from docs.

https://bugs.python.org/issue42392

### CODE REVIEW

**Comment by uriyyo** [comment]
@1st1 @asvetlov Can you please add `skip-news` label?

**Comment by uriyyo** [comment]
> Deprecated since version 3.8, removed in version 3.10: The loop parameter.

@1st1 @asvetlov I have found information in docs that `asyncio.Task` `loop` parameter is deprecated, [link](https://docs.python.org/3.10/library/asyncio-task.html#asyncio.Task) to the docs.

Is this a mistake in docs or there were no deprecation warning regarding `loop` parameter in `asyncio.Task` class?

**Comment by asvetlov** [comment]
It is a soft-deprecation.

Ideally, a user should never create a task by the constructor call but always use `loop.create_task()` or `asyncio.create_task()`.

Not sure what should we do with docs here.

File: Doc/library/asyncio-stream.rst [hunk 1]
```diff
@@ -49,8 +49,8 @@ and work with streams:
 
 
 .. coroutinefunction:: open_connection(host=None, port=None, \*, \
-                          loop=None, limit=None, ssl=None, family=0, \
-                          proto=0, flags=0, sock=None, local_addr=None, \
+                          limit=None, ssl=None, family=0, proto=0, \
```
Comment (asvetlov) [inline Doc/library/asyncio-stream.rst:52]: Drop `The *loop* argument is optional...` sentence below and for all four functions.

**Review by asvetlov — CHANGES_REQUESTED** [review]

File: Doc/library/asyncio-stream.rst (hunk 1 above)
Comment (uriyyo) [reply Doc/library/asyncio-stream.rst:52]: Done

**Comment by uriyyo** [comment]
I have made the requested changes; please review again

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**